src/jvm/main/org/jetbrains/kotlinx/lincheck/paramgen/DoubleGen.kt

```diff
@@ -0,0 +1,69 @@
+/*
+* #%L
+* Lincheck
+* %%
+* Copyright (C) 2015 - 2018 Devexperts, LLC
+* %%
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Lesser Public License for more details.
+*
+* You should have received a copy of the GNU General Lesser Public
+* License along with this program.  If not, see
+* <http://www.gnu.org/licenses/lgpl-3.0.html>.
+* #L%
+*/
+package org.jetbrains.kotlinx.lincheck.paramgen
+
+import java.util.*
+
+class DoubleGen(configuration: String) : ParameterGenerator<Double> {
+    private val random = Random(0)
+    private var begin = 0.0
+    private var end = 0.0
+    private var step = 0.0
+    override fun generate(): Double {
```

Comment:
please add an empty line between var-s and the function + follow this order by default:
1. fields
2. init block
3. functions
4. companion object